share/jupyter/nbconvert/templates/classic/base.html.j2

```diff
@@ -1,5 +1,5 @@
-{%- extends 'display_priority.j2' -%}
-{% from 'celltags.j2' import celltags %}
+{%- extends 'base/display_priority.j2' -%}
+{% from 'base/celltags.j2' import celltags %}
```

Comment:
Why is it necessary to refer to base here?

Reply:
Actually, if these changes in the jinja templates are required, this means this is really backward incompatible.